cms/templates/js/course-outline.underscore

```diff
@@ -201,20 +201,13 @@ if (is_proctored_exam) {
               </div>
             <% } %>
             <% if (xblockInfo.get('highlights_enabled') && course.get('self_paced') && xblockInfo.isChapter()) { %>
-              <div class="block-highlights">
-                    <span class="sr block-highlights-label"><%- gettext('Highlights:') %></span>
+                <div class="block-highlights">
                     <% var number_of_highlights = (xblockInfo.get('highlights') || []).length; %>
-                    <% if (number_of_highlights > 0) { %>
                         <span class="block-highlights-value highlights-button action-button">
-                            <%- edx.StringUtils.interpolate(
-                                gettext('Section Highlights: {number_of_highlights} entered'),
-                                {number_of_highlights: number_of_highlights}
-                            ) %>
+                            <span class="number-highlights"><%- number_of_highlights %></span>
```

Comment:
I just tested this with the Orca screen-reader & Firefox and this span is not accessible by tabbing through the UI. Changing this to a button makes is focus-able. We can do that in a separate a11y PR.